### PR TITLE
Add IAM user and policy

### LINKED ISSUE #9: create IAM roles and policies

### PR DESCRIPTION
**Pull request by jantillidie**
Closes #9

### CODE REVIEW

File: terraform/iam.tf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          "application-autoscaling:*",
+          "cloudwatch:*",
+          "dynamodb:*",
+          "ecs:*",
+          "acm:*",
+          "elasticloadbalancing:*",
+          "sns:*"
+        ]
+        Effect   = "Allow"
+        Resource = "*"
```
Comment (fabianschmauder) [inline terraform/iam.tf:35]: best practice would be to specify the resource a little bit more.

Maybe split the statement also for different services.

But for capstone also fine now :)

File: terraform/outputs.tf [hunk 2]
```diff
@@ -1 +1,8 @@
+output "access_key" {
+  value = aws_iam_access_key.tf_user_key.id
+}
 
+output "secret_key" {
+  value     = aws_iam_access_key.tf_user_key.secret
+  sensitive = true
+}
```
Comment (fabianschmauder) [inline terraform/outputs.tf:8]: why do you need the credentials as output?

**Review by fabianschmauder — COMMENTED** [review]
credentials as output seems strange to me. If you no really need it remove it

File: terraform/iam.tf (hunk 1 above)
Comment (jantillidie) [reply terraform/iam.tf:35]: When I use a sandbox, the resource changes every time, or not?
Will leave it as it is for now, have to look more into the topic.

**Pull request merged**